docs/changelog.mdx

```diff
@@ -5,42 +5,55 @@ cardHeadline: Changelog
-* conflicts between next and main v2 ([#281](https://github.com/freenowtech/wave/issues/281)) ([bec6bdc](https://github.com/freenowtech/wave/commit/bec6bdc376fa5ae009e4c56adec86148b3105be3))
+-   adding disabled state for datepicker ([#181](https://github.com/freenowtech/wave/issues/181)) ([313f424](https://github.com/freenowtech/wave/commit/313f4240190767f77bb00340f4ecbbb8e215a2af))
```

Comment:
For some reason the formatting of the changelog changed at some point, this is just making the new entries the same format as the old ones. 
As for the weird entries in `2.1.0-next.2`, I'd let them be for now, we'll probably have to manually create the entire changelog for `next` since creating it automatically failed due to failed jobs, but I preferred not doing it in this PR